src/store/user/reducer.js

```diff
@@ -0,0 +1,27 @@
+import { DELETE_USER, SAVE_USER } from './types';
+
+export const userInitialState = {
+	isAuth: false, // default value - false. After success login - true
+	name: '', // default value - empty string. After success login - name of user
+	email: '', // default value - empty string. After success login - email of user
+	token: '', // default value - empty string or token value from localStorage.
+	// After success login - value from API /login response. See Swagger.
+};
+
+// Use the initialState as a default value
+export const userReducer = (state = userInitialState, action) => {
+	// The reducer normally looks at the action type field to decide what happens
+	switch (action.type) {
+		// Do something here based on the different types of actions
+		case SAVE_USER:
```

Comment:
here's where you can handle your auth variable.

case SAVE_USER: {

return {
...state,
isAuth: true,
...action.payload,
}

}

Reply:
👍